pkg/helm/handlers/handlers.go

```diff
@@ -299,3 +352,11 @@ func (h *helmHandlers) HandleIndexFile(user *auth.User, w http.ResponseWriter, r
 
 	w.Write(out)
 }
+
+func NewCoreClient(conf *action.Configuration) (*corev1client.CoreV1Client, error) {
```

Comment:
Let's move `NewCoreClient` to another file in the same package, probably together with the `DynamicClient` function.

Reply:
Done